modules/masterbar/admin-color-schemes/class-admin-color-schemes.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Admin color schemes file.
+ *
+ * @package Jetpack
+ */
+
+namespace Automattic\Jetpack\Dashboard_Customizations;
+
+/**
+ * Unifies admin color scheme selection across WP.com sites.
+ */
+class Admin_Color_Schemes {
+
+	/**
+	 * Admin_Color_Schemes constructor.
+	 */
+	public function __construct() {
+		add_action( 'rest_api_init', array( $this, 'register_admin_color_meta' ) );
+	}
+
+	/**
+	 * Makes admin_color available in users REST API endpoint.
+	 */
+	public function register_admin_color_meta() {
+		register_meta(
+			'user',
+			'admin_color',
+			array(
+				'default'       => 'fresh',
+				'description'   => __('Slug of the admin color scheme.', 'jetpack'),
+				'single'        => true,
+				'show_in_rest'  => current_user_can('read'),
```

Comment:
```suggestion
				'show_in_rest'  => current_user_can( 'read' ),
```

Reply:
It doesn't look like we can limit `show_in_rest` to only be true for "self requests". But we can hide the value. Would that be worth doing?
```suggestion
				'show_in_rest'  => array(
					'prepare_callback' => function( $value, $request ) {
						if ( current_user_can( 'edit_user', $request->get_param( 'id' ) ) ) {
							$value = '';
						}
						return $value;
					},
				),
```

#### Result:
![image](https://user-images.githubusercontent.com/1398304/98976414-2c4b6900-24cc-11eb-8a23-c5c37b5f9b2e.png)

Reply:
I switched it back to `true` to surface the meta key to unauthenticated OPTIONS requests. @jeherve @getdave Any vetos on that or do you think that's fine?